Author: Andreas Beckmann <anbe@debian.org>
Description: (HACK!) fix ARCH detection for armhf running on arm64
 since the 390xx series does not support arm64 at all, just map
 armv8l to arm, too, to fix building the kernel module in armhf
 chroots running on arm64 hosts

--- a/Makefile
+++ b/Makefile
@@ -54,6 +54,7 @@ else
   ifndef nvkARCH
     nvkARCH := $(shell uname -m | sed -e 's/i.86/i386/' \
       -e 's/armv[0-7]\w\+/arm/' \
+      -e 's/armv8l/arm/' \
       -e 's/aarch64/arm64/' \
       -e 's/ppc64le/powerpc/' \
     )
